net-libs/libwebsockets/metadata.xml

```diff
@@ -19,12 +19,10 @@
 		libwebsockets is a C library designed to make development of websockets clients and servers easier (possible).
 	</longdescription>
 	<use>
-		<flag name="ssl">Include SSL support (default <pkg>dev-libs/openssl</pkg>)</flag>
-		<flag name="zlib">Include zlib support</flag>
-		<flag name="static-libs">Build the static version of the library</flag>
+		<flag name="client">The client part of the library</flag>
```

Comment:
So let's say:
`The client part of the library and libwebsockets-test-client`
And set
`LWS_WITHOUT_TEST_CLIENT` the same value as `LWS_WITHOUT_CLIENT`  ?